aspnet/tutorials/web-api-help-pages-using-swagger.rst

```diff
@@ -0,0 +1,328 @@
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs
+    :language: c#
+    :start-after: snippet_Configure
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 16-26
+
+Then the Swagger UI produces the following more presentable information concerning the API.
+
+.. image:: web-api-help-pages-using-swagger/_static/custom-info.png
+
+XML Comments
+'''''''''''''
+In order to enable the generation of xml comments, right click the project in Visual Studio and select "Properties" and check the ``XML Documentation file`` box under the Output Settings section.
+
+.. image:: web-api-help-pages-using-swagger/_static/swagger-xml-comments.png
+
+.. note:: If you are not using Visual Studio then you can enable the "XML documentation file" flag in the ``project.json`` file by setting the ``"xmlDoc": true`` in the ``buildOptions`` section.
+
+Configure the OperationFilter to use the generated xml file. An additional item to point out here is the use of ``PlatformServices.Default.Application.ApplicationBasePath`` to access the executing location of the application. This is the location where the generated xml file will be placed. Notably it can vary depending on the .NET Framework used for the environment.
+
+.. note:: For Linux or non-Windows operating systems, file names and paths can be case sensitive. So ``ToDoApi.xml`` would be found on Windows but not Centos for example.
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Startup.cs
+    :language: c#
+    :start-after: snippet_Configure
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 29,32
+
+Adding the triple slash comments to the method enhances the Swagger UI by adding the description to the header of the section.
+
+.. literalinclude:: web-api-help-pages-using-swagger/sample/src/TodoApi/Controllers/TodoController.cs
+    :language: c#
+    :start-after: Delete_Method
+    :end-before: #endregion
+    :dedent: 8
+    :emphasize-lines: 2
+
+.. image:: web-api-help-pages-using-swagger/_static/triple-slash-comments.png
+
+Note that the UI is driven by the generated JSON file, and these comments are also in that file as well.
+
+.. code-block:: javascript
+    :emphasize-lines: 5
+
+      "delete": {
+        "tags": [
+          "Todo"
+        ],
+        "summary": "Deletes a specific TodoItem",
+        "operationId": "ApiTodoByIdDelete",
+        "consumes": [],
+        "produces": [],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No Content"
+          }
+        },
+        "deprecated": false
+      }
+
+Here is a more robust example, adding ``<remarks />`` where the content can be just text or adding the JSON or xml object for further documentation of the method.
```

Comment:
I think I get it - the  SwaggerResponse   has nothing to do with it. As you can see this is very confusing - so the snippet can't include extra stuff